Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -171,6 +173,21 @@ export default class VerifySMSComponent extends Vue {
             });
     }
 
+    private getTimeout(): number {
+        if (!this.smsResendDateTime) {
+            this.updateSMSResendDateTime({
+                hdid: this.user.hdid,
+                dateTime: new DateWrapper(),
+            });
+        }
+        let resendTime = new Date();
+        resendTime.setTime(
+            this.smsResendDateTime!.toJSDate().getTime() +
```

Comment:
nit: i think there is a method to get the millis since the epoch

Reply:
this is said function! quote "The .getTime() call retrieves the milliseconds since epoch in UTC."

Reply:
I meant that  there is no need to convert to Date, DateWrapper to millis should be a function

Reply:
changed :)